examples/tests/org.jboss.tools.project.examples.test/src/org/jboss/tools/project/examples/tests/CheatSheetTest.java

```diff
@@ -130,7 +130,7 @@ private static void importProject(IProgressMonitor monitor) throws MalformedURLE
 		Collection<ProjectExample> projects = ProjectExamplesActivator.getDefault().getProjectExampleManager().getExamples(monitor);
 		ProjectExampleWorkingCopy projectExample = null;
 		for (ProjectExample project: projects) {
-			if (KITCHENSINK.equals(project.getName())) {
+			if ("kitchensink".equals(project.getName())) {
```

Comment:
ok, makes sense. I think another constant to clearly distinguish project name and folder name would make sense, though.